File: GLLara/GLLDrawingPreferencesViewController.m
//
//  GLLDrawingPreferencesViewController.m
//  GLLara
//
//  Created by Torsten Kammer on 29.04.18.
//  Copyright © 2018 Torsten Kammer. All rights reserved.
//

#import "GLLDrawingPreferencesViewController.h"

#import "GLLara-Swift.h"

@interface GLLDrawingPreferencesViewController ()

@property (nonatomic, readwrite, assign) NSUInteger maxAnisotropyLevel;

@end

@implementation GLLDrawingPreferencesViewController

- (id)init {
    if (!(self = [super initWithNibName:@"GLLDrawingPreferencesView" bundle:nil])) {
        return nil;
    }
    
    return self;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.maxAnisotropyLevel = [[GLLResourceManager shared] maxAnisotropyLevel];
}

@end

File: GLLara/GLLDrawingPreferencesViewController.swift
//
//  GLLDrawingPreferencesViewController.swift
//  GLLara
//
//  Created by Torsten Kammer on 23.07.22.
//  Copyright © 2022 Torsten Kammer. All rights reserved.
//

import Foundation

@objc class GLLDrawingPreferencesViewController: NSViewController {
    
    convenience init() {
        self.init(nibName: "GLLDrawingPreferencesView", bundle: nil)
    }
    
    @objc dynamic var maxAnisotropyLevel: Int {
        return GLLResourceManager.shared.maxAnisotropyLevel
    }
}
